SetGame/SetGameModel.swift

```diff
@@ -31,16 +31,26 @@ class SetGameModel {
 		checkMatching()
 	}
 
+	//TODO test
```

Comment:
Would it be reasonable to have at least a smoke test for this change?

Reply:
sure